src/_11ty/filters/formatDate.js

```diff
@@ -1,4 +1,7 @@
-module.exports = (date, locale = site.defaultLocale) => {
+const site = require('../../_data/site');
+const { defaultLocale } = site;
+
+module.exports = (date, locale = defaultLocale) => {
```

Comment:
Good catch. Didn't realise global data isn't available to filters. Perhaps the v1 of 11ty might have neater ways of getting access to that, or we can look at passing it along from the 11ty config, so these filters remain pure.

Reply:
@adamduncan I done a little research and couldn't find anything relating to making global data available to filters.

It does seem like [v1 will allow global data to be added to the 11ty config](https://www.11ty.dev/docs/data-global-custom/) but I still can't find any mention of this being available to filters.

Let's revisit this once/if we upgrade to v1.

Reply:
🤔 I guess it maybe possible to pass along the `defaultLocale` if we added the filter directly to the config file rather than requiring it.

So this syntax:
```
eleventyConfig.addFilter("formatDate", function(value) { … });
```

rather than
```
eleventyConfig.addFilter('formatDate', require(`${filtersDir}/formatDate.js`));
```